gradle.properties

```diff
@@ -19,3 +19,8 @@ android.useAndroidX=true
 android.enableJetifier=true
 # Kotlin code style for this project: "official" or "obsolete":
 kotlin.code.style=official
+#Android Studio and Gradle are using different locations for the JDK.
```

Comment:
Don't upload this

Reply:
it's only a comment to be able to switch quickly between branches, otherwise I've to change manually 3 files everty time i create a new folder,